activestorage/app/controllers/active_storage/blobs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# Take a signed permanent reference for a blob and turn it into an expiring service URL for download.
```

Comment:
How are these four controllers in app/controllers/active_storage  intended to be used in a Rails app?
Are they meant as superclasses to inherit from?